api/graphql/index.js

```diff
@@ -109,7 +110,8 @@ function createSchema (userId, isAdmin) {
       deletePost: (root, { id }) =>
         deletePost(userId, id),
       addSkill: (root, { name }) => addSkill(userId, name),
-      removeSkill: (root, { name }) => removeSkill(userId, name)
+      removeSkill: (root, { name }) => removeSkill(userId, name),
+      removeMember: (root, {personId, communityId}) => removeMember(personId, communityId, userId)
```

Comment:
small thing, but I think it would make sense to have `userId` be the first param to `removeMember`, just for consistency with the other mutations